### DISCUSSION TITLE
Question about formula of x_0(length of near wake) in gauss deflection model

### DISCUSSION THREAD

**Discussion by xxxiiinn** [NatLabRockies/floris#278] [Q&A]
Hi everyone,

I have look over the code of gauss deflection model in "gauss.py" file, and I found that the formula used to calculate the length of near wake x_0 from line 165 to 170 is:

` 
 x0 = (
            D
            * (cosd(yaw) * (1 + np.sqrt(1 - Ct * cosd(yaw))))
            / (np.sqrt(2) * (4 * alpha * TI + 2 * beta * (1 - np.sqrt(1 - Ct))))
            + coord.x1
        )
`
Comparing to the formula in the literature of Bastankhah:

![image](https://user-images.githubusercontent.com/28628919/150059573-c3db1864-ca77-4e75-9023-5d4b5fcb9090.png)

An additional cos(yaw) is in numerator with Ct. I want to know the reason for this additional term, or maybe it's just a code-editing error.

Xin

**Comment by Bartdoekemeijer** [comment]
Hi Xin,

I haven't gone into the mathematics, but wanted to remark something that might be useful in this discussion. `Ct` is the thrust coefficient, thus the thrust that the rotor applies on the flow normalized by rotor and flow properties. Now, `Ct` has a clear definition in situations where the rotor is perfectly aligned with the ambient flow.
However, when the rotor is misaligned, different definitions of `Ct` arise. For example, the thrust vector can be the force parallel to the flow, or it can be the vector orthogonal to the rotor plane. Both definitions have been used in the literature, yet are often not explicitly clarified. Though, these quantities differ by cos(gamma). I think the same questions arises with which wind speed is used for normalization. From what I recall, is that the definition for `Ct` used in Bastankhahs article differs from how it is defined in FLORIS, and that could perhaps explain the additional `np.cos(gamma)` term. Though, probably the other FLORIS developers can provide a better insight here.

Best,
Bart

**Reply by xxxiiinn** [reply]
Hi Bart，

Thanks for you reply.

From what I found, the CT used in Bastankhahs article is already include the effect of yaw angle. This is the same definition of CT used in Floris turbine class.

![image](https://user-images.githubusercontent.com/28628919/160745760-3be94acb-542e-4791-b39a-68549ba2a13f.png)

So, if we comparing the fomula used to calculate U_R, we will find  Floris and Bastankhahs article are same.

![image](https://user-images.githubusercontent.com/28628919/160748491-6a6199f6-16ef-42ab-a9d7-e6a3d493e7fa.png)

![image](https://user-images.githubusercontent.com/28628919/160748524-2a5dd51f-cbc7-4052-aac0-9ea3e0c2900b.png)

But as I mentioned before, the formula used to calculate the length of near wake x_0 are different. 

Best,
Xin
